### PR TITLE
Recs event collection clarifications 

### PR DESCRIPTION
**Pull request by misha-kotov**
## Purpose of this pull request

Adds information about accepting cookies to ensure event collection is successful.

## Affected DevDocs pages

-  https://devdocs.magento.com/recommendations/verify.html

### CODE REVIEW

File: src/recommendations/verify.md [hunk 1]
```diff
@@ -22,6 +22,7 @@ To ensure that the `DataServices.js` file is loading on all site pages:
 {:.procedure}
 To ensure events are firing on pages across your site (home, product, checkout, and so on):
 
+1. Make sure you disable any ad blockers on your browser and accept cookies on the site.
```
Comment (BarnyShergold) [inline src/recommendations/verify.md:25]:
```suggestion
1. Make sure you disable any ad blockers on your browser and accept cookies on the site.
```

File: src/recommendations/verify.md [hunk 2]
```diff
@@ -34,7 +35,7 @@ To ensure events are firing on pages across your site (home, product, checkout,
 
 Install the [Snowplow Analytics Debugger extension for Chrome](https://chrome.google.com/webstore/detail/snowplow-analytics-debugg/jbnlcgeengmijcghameodeaenefieedm). This extension displays the events being collected and sent to Magento.
 
-1. Make sure you disable any ad blockers on your browser.
+1. Make sure you disable any ad blockers on your browser and accept cookies on the site.
```
Comment (BarnyShergold) [inline src/recommendations/verify.md:38]:
```suggestion
1. Make sure you disable any ad blockers on your browser and accept cookies on the site.
```

**Review by BarnyShergold — CHANGES_REQUESTED** [review]
Please add a more meaningful PR title - "Small Changes" is not sufficient. Also complete the PR description as required by the guidelines. Check an accepted review for an example

You had 2 linting errors - accepting the two small suggested changes should fix this.

**Comment by BarnyShergold** [comment]
@misha-kotov - Thanks for some of the updates but you still need to create a proper PR description before the PR can be approved

**Review by erikmarr — APPROVED** [review]

**Comment by erikmarr** [comment]
running tests

**Pull request merged**